src/interpreter/constructs.ts

```diff
@@ -620,13 +622,15 @@ export const RETURN_CONSTRUCTS: {
     ): ConstructResult => {
       if (node.text === 'input') {
         if (!inputStructure || !inputStructure.fields) {
+          const accessKey = getAccessKey(node);
+
           return returnIssue(
             {
               kind: 'useCaseSlotNotFound',
               context: {
                 path: getPath(node),
                 expected: UseCaseSlotType.INPUT,
-                actual: getVariableName(node),
+                actual: isAccessKeyError(accessKey) ? accessKey.message : accessKey.key,
```

Comment:
question: As we talked about this irl - why not do a static message here in the end here? Won't it always be `"input"` here anyway?

Reply:
I returned it because the error message looked better with full path. Here is how it is formatted https://github.com/superfaceai/parser/blob/fix/enum-validation-2/src/interpreter/utils.ts#L143-L147